common-lib/src/cmr/common/api/errors.clj

```diff
@@ -27,8 +27,8 @@
    :body {:errors ["An Internal Error has occurred."]}})
 
 (defn- keyword-path->string-path
-  "Converts a set of keyword field paths into the string equivalent field paths to return to the
-  user."
+  "Converts a set of keyword field paths into the string equivalent field paths
```

Comment:
*laughs* Oh, man ... too good :-)

Yeah, it looks like bikeshed lets you configure line length checks with `--max-line-length`.